mixins/README.md

```diff
@@ -0,0 +1,49 @@
+# Terraform Mixins
+
+A Terraform mixin (inspired by the [concept of the same name in OOP languages such as Python and Ruby](https://en.wikipedia.org/wiki/Mixin))
+is a Terraform configuration file that can be dropped into a root-level module, i.e. a component, in order to add additional
+functionality.
+
+Mixins are meant to encourage code reuse, leading to more simple components with less code repetition between component
```

Comment:
```suggestion
Mixins are meant to encourage code reuse, leading to more simple components with less code repetition between components
```

Reply:
The README is auto-generated, so I will edit the inline doc.